debunkbot/utils/claims_handler.py

```diff
@@ -41,16 +41,19 @@ def fetch_claims_from_gsheet():
     google_sheet_helper = GoogleSheetHelper()
     total_claims = 0
     for claim_database in claim_databases:
+        RATINGS = list(map(lambda x: x.upper(), claim_database.claims_ratings))
```

Comment:
```suggestion
        ratings = list(map(lambda x: x.upper(), claim_database.claims_ratings))
```
Why are we capitalizing it?

Reply:
There is also this comment @esirK